test/replset_tests.js

```diff
@@ -54,16 +54,15 @@ describe('ReplSet', function() {
         assert.ok(typeof version.version[0] === 'number');
         assert.ok(typeof version.version[1] === 'number');
         assert.ok(typeof version.version[2] === 'number');
-        done();
       }).catch(function(err) {
```

Comment:
you'll want to remove the `catch` here for all of these, since mocha will automatically catch the errors and print a useful stack trace.

On further review it looks like you might have just missed one or two here